include/account/account.h

```diff
@@ -4,16 +4,27 @@
 #include "address.h"
 #include <string>
 
+#ifndef DEFAULT_NONCE
+    #define DEFAULT_NONCE 0U
+#endif
+
+#define DEFAULT_BALANCE "0"
+
 class Account
 {
 public:
     explicit Account(Address address);
 
+    explicit Account(Address address, std::string balance, uint64_t const &nonce);
+
     const Address& getAddress() const;
 
     const std::string& getBalance() const;
 
+    const uint64_t& getNonce() const;
```

Comment:
`incrementNonce` will also be quite useful (e.g. see erdpy and erdjs).

Reply:
Good point. Implement it + added gtests as well